spot_driver/spot_driver/spot_ros2.py

```diff
@@ -201,6 +203,21 @@ def set_node_parameter_from_parameter_list(
 class SpotROS(Node):
     """Parent class for using the wrapper.  Defines all callbacks and keeps the wrapper alive"""
 
+    """
```

Comment:
While I like this method, I wonder if it's a good idea to expose these parameters in the  SpotROS2 constructor

Reply:
Sorry, I don't understand this comment.

Reply:
Do you mean these parameters here?
    TRAJECTORY_BATCH_SIZE = 100
    TRAJECTORY_BATCH_OVERLAPPING_POINTS = 20

Reply:
Done. I converted the constants into ROS parameters with default values.